pkg/assessor/manifest/manifest.go

```diff
@@ -19,6 +23,7 @@ type ManifestAssessor struct{}
 
 var ConfigFileName = "metadata"
 var (
+	sensitiveWords   = []string{"pass", "passwd", "access", "token", "secret", "key", "api", "password"}
```

Comment:
I think we could reuse `suspiciousEnvKey` instead of creating another set of words, both look for sensitive environment variables names.

Reply:
Removed sensitiveWords and now only using suspiciousEnvKey